src/Schema/Values/OwnerType.ts

```diff
@@ -40,6 +40,14 @@ export enum OwnerType {
   myCollection = "myCollection",
   myCollectionArtwork = "myCollectionArtwork",
   onboarding = "onboarding",
+  ordersAccept = "orders-accept",
+  ordersCounter = "orders-counter",
+  ordersNewPayment = "orders-new-payment",
+  ordersOffer = "orders-offer",
+  ordersPayment = "orders-payment",
+  ordersRespond = "orders-respond",
+  ordersReview = "orders-review",
+  ordersShipping = "orders-shipping",
```

Comment:
We are using these kebab-case values in the hardcoded force values, but everything else in this file is using camelCase. Is that a problem???

Reply:
This is great and it is consistent with what we've been doing!